frontend/lobby/splash.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as movember from './images/funded-by-movember.png';
+
+// Include our SCSS (via webpack magic)
+import './lobby.scss';
```

Comment:
I think we only need this in one file, so the existing line in `lobby.tsx` should be fine.